bagelbids/cli.py

```diff
@@ -122,3 +124,40 @@ def pheno(
 
     with open(output / "pheno.jsonld", "w") as f:
         f.write(json.dumps(context, indent=2))
+
+
+@bagel.command()
+def add_bids(
+    jsonld_path: Path = typer.Option(
+        ...,
+        help="The path to a pheno.jsonld file.",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+    ),
+    bids_dir: Path = typer.Option(
+        ...,
+        help="The path to the corresponding BIDS dataset directory.",
+        exists=True,
+        file_okay=False,
+        dir_okay=True,
+    ),
+    output: Path = typer.Option(
+        ...,
+        help="The directory where outputs should be created",
+        exists=True,
+        file_okay=False,
+        dir_okay=True,
+    ),
+):
+    jsonld = load_json(jsonld_path)
+    layout = BIDSLayout(bids_dir, validate=True)
+
+    context = jsonld.pop("@context")
```

Comment:
I feel like this line could use a comment on why we need to pop the context off: because it isn't part of the model (and because we can't easily add the context to the model)